src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -62,6 +62,11 @@ internal void SetErrors(IEnumerable<IDocumentError> errors)
         /// </summary>
         public ParseResult Parse { get; set; }
 
+        /// <summary>
+        /// Symbols passed to this binding. May be null. 
+        /// </summary>
+        public ReadOnlySymbolTable Symbols { get; set; }

```

Comment:
>set [](http://example.com/codeflow?start=50&length=3)

oof: just noticed on this class, but all these `set` should be `internal set` or `internal init`

Reply:
actually, not necessarily. 
ASP.Net follows this pattern with settable properties - it can be convenient for the caller and testing.